### ISSUE TITLE
Purgeable debuffs override nameplate color changes

### ISSUE THREAD

**Issue by jcadduono** [rljohn/kuinameplates-target-helper#13]
Regardless of plugin priority, once an enemy has a purgeable debuff / enrage effect, their health bars return to original Kui Nameplates color instead of the color selected for debuffs active in knp Target Helper

**Comment by BuurmanDeBuurman** [comment]
Or has an absorb. Having the same issue.

**Comment by rljohn** [comment]
I'll take a look after the holidays

**Comment by jcadduono** [comment]
Any luck with this? Very frustrating on Raging week in keys on a Druid

**Comment by rljohn** [comment]
I can investigate tonight. Any suggestions of a good place I could test with in isolation?

**Comment by jcadduono** [comment]
I suppose if you have a priest (dispel magic), DH (consume magic), or shaman (purge) you can hit up the elite Fire Elementals in eastern Azure Span, they put a shield on themselves every 10 seconds or so I think
It's where the world boss Bazual is up right now

**Comment by rljohn** [comment]
Thank you, I'll see what I can do

**Comment by rljohn** [comment]
Latest version should fix this, let me know if it works for you.

**Comment by jcadduono** [comment]
yeah it works great sorry it took a while to reply, I quit playing the feral druid for a bit
